internal/quickstart/messages.go

```diff
@@ -40,7 +40,12 @@ func toggleFlag(client flags.Client, accessToken, baseUri, flagKey string, enabl
 			flags.BuildToggleFlagPatch(defaultEnvKey, enabled),
 		)
 		if err != nil {
-			return errMsg{err: err}
+			var e struct {
```

Comment:
Can you make this show a nicer error message on conflict like the `createFlag` command? And then could you refactor the duplication to a single place?

Reply:
Oh sure. What would the conflict on a flag toggle look like? Would it be if the flag is already targeted on and we're target it on with our api call?

Reply:
Try toggling it a bunch to see the error show up.